### PR TITLE
Support for docker compose v2

### PR DESCRIPTION
**Pull request by hrubi**

### CODE REVIEW

**Review by bossek — APPROVED** [review]

File: mix.exs [hunk 1]
```diff
@@ -5,7 +5,7 @@ defmodule DockerCompose.MixProject do
     [
       app: :docker_compose,
       version: "0.3.1",
-      elixir: "~> 1.10",
+      elixir: "~> 1.15",
```
Comment (Coffei) [inline mix.exs]: Do we need to require Elixir 1.15? For a library it's generally better to keep this requirement as loose as possible. Did you take advantage of any 1.15 features?

Comment (hrubi) [reply mix.exs]: I did not. I will keep it at `1.10` then.

Comment (hrubi) [reply mix.exs]: Force-pushed. [Diff](https://github.com/altworx/docker-compose-elixir/compare/d29dc1ddb35d24b2dc2f31a02836e62a43ddc025..71679bcaeffcfbe0ae79f435a5a171cb784ffd23).

Comment (Coffei) [reply mix.exs]: Thanks!

**Review by Coffei — APPROVED** [review]

**Pull request merged**